src/components/SwipeClassifier.js

```diff
@@ -2,50 +2,94 @@ import React from 'react'
-import { startNewClassification } from '../actions/classifier'
+import { startNewClassification, setTutorialCompleted } from '../actions/classifier'
+import { isEmpty } from 'ramda'
 
 const mapStateToProps = (state, ownProps) => ({
   isFetching: state.classifier.isFetching,
   workflow: state.classifier.workflow[ownProps.workflowID] || {},
+  project: state.classifier.project[ownProps.workflowID] || {},
+  tutorial: state.classifier.tutorial[ownProps.workflowID] || {},
+  needsTutorial: state.classifier.needsTutorial[ownProps.workflowID] || false,
 })
 
 const mapDispatchToProps = (dispatch) => ({
   startNewClassification(workflowID) {
     dispatch(startNewClassification(workflowID))
   },
+  setTutorialCompleted() {
+    dispatch(setTutorialCompleted())
+  },
   setIsFetching(isFetching) {
     dispatch(setState('classifier.isFetching', isFetching))
   },
 })
 
 export class SwipeClassifier extends React.Component {
-  componentWillMount() {
+  constructor(props) {
+    super(props)
+    this.setQuestionVisibility = this.setQuestionVisibility.bind(this)
+    this.state = {
+      isQuestionVisible: true,
+    }
     this.props.setIsFetching(true)
     this.props.startNewClassification(this.props.workflowID)
   }
 
+  setQuestionVisibility(isVisible) {
+    this.setState({isQuestionVisible: isVisible})
+  }
+
+  finishTutorial() {
+    if (this.props.needsTutorial) {
+      this.props.setTutorialCompleted()
+    } else {
+      this.setQuestionVisibility(true)
+    }
+  }
+
   static renderNavigationBar() {
     return <NavBar title={'Classify'} showBack={true} />;
   }
 
   render() {
-    const renderClassifier = () => {
+    const renderClassifierOrTutorial = () => {
       const key = this.props.workflow.first_task //always just one task
       const task = this.props.workflow.tasks[key]
-      return (
+
+      const classifier =
+        <Question question={task.question} workflowID={this.props.workflowID} />
+
+      const tutorial =
+        <Tutorial
+          projectName={this.props.project.display_name}
+          isInitialTutorial={this.props.needsTutorial}
+          tutorial={this.props.tutorial}
+          finishTutorial={() => this.finishTutorial()} />
+
+      const classifierOrTutorial = this.state.isQuestionVisible ? classifier : tutorial
+
+      const classificationPanel =
         <ClassificationPanel
           isFetching={ this.props.isFetching }
-          hasTutorial = { false }>
-          <Question question={task.question} workflowID={this.props.workflowID} />
+          hasTutorial = { !isEmpty(this.props.tutorial) }
+          isQuestionVisible = {this.state.isQuestionVisible }
+          setQuestionVisibility = { this.setQuestionVisibility }>
+          { classifierOrTutorial }
         </ClassificationPanel>
+
+      return (
+        this.props.needsTutorial ? tutorial : classificationPanel
       )
     }
 
+    const renderSpinner = this.props.isFetching || this.props.workflow === undefined
```

Comment:
It ought not get there because of that catch which give an alert error and sends them back, but I don't think that was needed at any rate  (I added it while testing something else)  so I've removed it.